subsys/net/lib/lwm2m/lwm2m_rw_cbor.c

```diff
@@ -0,0 +1,574 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_MODULE_NAME net_lwm2m_cbor
+#define LOG_LEVEL CONFIG_LWM2M_LOG_LEVEL
+
+#include <logging/log.h>
+#include <sys/util.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include "cbor_common.h"
+#include "cbor_decode.h"
+#include "cbor_encode.h"
```

Comment:
Are those public headers? Anyway, they should use `<>` includes as compiler should search inlcude paths for those.

Reply:
Done. Yes, those are cddl-gen (zcbor) public headers.